instrumentation/manticore/lib/opentelemetry/instrumentation/manticore/util/wrapped_request.rb

```diff
@@ -0,0 +1,50 @@
+module OpenTelemetry
```

Comment:
Is wrapping the request/response necessary in this way? My quick look at the manticore gem made it seem like `request.header` and/or `response.header` was just a hash, so I'm not sure what we gain here?

Reply:
I removed the WrappedResponse class, but the WrappedRequest class is required due to the propagation required to insert into the headers. The propagation uses `[]=`. 

Reference: https://github.com/schanjr/opentelemetry-ruby/blob/feat_add_manticore_instrumentation/instrumentation/manticore/lib/opentelemetry/instrumentation/manticore/patches/response.rb#L24 

There's also some convenient methods I added as well.